modal for add emoji alias

songgao opened the pull request.

hyperobject commented on shared/util/emoji.tsx:
```diff
@@ -12,20 +13,13 @@ export type EmojiData = {
-        source: emoji.source.httpsrv,
-        unified: '',
-      }
+  return {
```
picker and autocomplete don't handle aliases well but that's probably another ticket

songgao replied:
I think with this change alias now works in the picker. I'll look into autocomplete

hyperobject commented on shared/teams/team/rows/emoji-row/add.tsx:
```diff
@@ -25,7 +25,17 @@ const AddEmoji = ({teamID, convID, filter, setFilter}: OwnProps) => {
       })
     )
-  const onAddAlias = () => {}
+  const onAddAlias = () =>
```
would be good to hook this up to the "add alias"/"edit alias" menu buttons in `emoji-row/item.tsx`

songgao commented:
@hyperobject Please take another look; thanks!

New changes:

* hooked up the alias modal to the item popup
* I don't think we are doing "Edit Alias" so I just removed it
* added the call for removing emoji
* reload user emojis after adding an emoji/alias, or removing an emoji
* fixed alias in autocompleter

I noticed the team sectionlist flashes when it reloads and I'm still looking into that.

songgao commented:
Alias to stock emoji doesn't render correctly on mobile yet. It's cut-off on top. Will visit again after https://github.com/keybase/client/pull/23533 is in.
